references/cli.md

```diff
@@ -277,6 +318,44 @@ Options:
   --help                    Show this message and exit.
 ```
 
+## substra add composite_traintuple
+
+```bash
+Usage: substra add composite_traintuple [OPTIONS]
+
+  Add composite traintuple.
+
+  The option --data-samples-path must point to a valid JSON file with the
+  following schema:
+
+  {
+      "keys": list[str],
+  }
+
+  Where:
+  - keys: list of data sample keys
+
+Options:
+  --objective-key TEXT            [required]
+  --algo-key TEXT                 [required]
+  --dataset-key TEXT              [required]
+  --data-samples-path FILE        [required]
+  --head-model-key TEXT           Must be used with --trunk-model-key option.
+  --trunk-model-key TEXT          Must be used with --head-model-key option.
+  --out-trunk-model-permissions-path FILE
+                                  Load a permissions file.
```

Comment:
Could you include in the general description an explanation of the contents of a "permissions file"?